ui_framework/components/color_picker/_color_picker.scss

```diff
@@ -0,0 +1,36 @@
+.kuiColorPicker {
+  cursor: pointer;
+}
+
+.kuiColorPicker__preview {
```

Comment:
Can we indent `kuiColorPicker__preview`, `kuiColorPicker__swatch`, `
 +.kuiColorPicker__label`? This visually communicates the parent-child relationship without incurring the specificity you get when you nest them.